contracts/mocks/MockCTKNCrowdsaleWithMutableDates.sol

```diff
@@ -0,0 +1,65 @@
+// solhint-disable no-empty-blocks
```

Comment:
Is this considered a Stub? I mix the words around, but sometime think of fakes and mocks as the same level, whereas this actual has real behavior

Reply:
I think its technically a 'fake' since it has a working implementation

Reply:
I only point it out because as I was reading the code I got a bit confused because I thought it was inheriting from an interface and not the actual CTKNCrowdsale.

Reply:
I don't make a distinction.